app/javascript/flavours/polyam/features/account_timeline/components/anniversary-note.tsx

```diff
@@ -0,0 +1,24 @@
+import { FormattedMessage } from 'react-intl';
+
+import type { Account } from 'flavours/polyam/models/account';
+
+export const AnniversaryNote: React.FC<{ account: Account }> = ({
+  account,
+}) => {
+  return (
+    <div className='account-anniversary-banner'>
+      <div className='account-anniversary-banner__message'>
+        <FormattedMessage
+          id='account.anniversay'
+          defaultMessage="It's {acct}'s {years, selectordinal, one {#st} two {#nd} few {#rd} other {#th}} fediversary!"
```

Comment:
Not sure how translatable this is

Reply:
Another concern I have about displaying the years is the "number go up effect", but since it's displayed only 1 day in a year it's probably fine and it doesn't give more information than is already available through the join date.

Reply:
Translations don't have to adhere to that format, so it works with languages not having this pattern.